pkg/step_if.go

```diff
@@ -58,6 +59,22 @@ func (l IfStepLoader) LoadStep(config StepDef, context LoadingContext) (Step, er
 	result.If = ifInput
 	result.Then = thenInput
 
+	var elseArray interface{}
+	elseData := config.Get("else")
+
+	if elseData != nil {
+		elseArray, ok = elseData.(interface{})
+		if !ok {
+			return nil, fmt.Errorf("field \"if\" must be an interface{} but it wasn't: %v", ifData)
```

Comment:
This should be `field \"else\"` must be ....`

Reply:
yes, bad copy-paste :)

Reply:
Please review: https://github.com/mumoshu/variant/pull/93